src/components/tooltip/tooltip.visualroute.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { css } from '@emotion/core';
+import { PrimaryButton, Tooltip } from 'ui-kit';
+import { Suite, Spec } from '../../../test/percy';
+
+const title = 'What kind of bear is best';
+const noop = () => {};
+
+export const routePath = '/tooltip';
+
+export const component = () => (
+  <Suite>
+    <Spec label="Closed" omitPropsList>
+      <Tooltip title={title} eventsEnabled={false}>
```

Comment:
Where is this prop `eventsEnabled` defined? 🤔

Reply:
ah yeah that's a leftover from trying to get poppers to work with percy. I'll remove it.

Reply:
Do you want to remove it?